tests/csm/rest/test_capacity_usage.py

```diff
@@ -850,7 +863,7 @@ def test_34718(self):
         self.log.info("##### Test started -  %s #####", test_case_name)
         self.log.info("Step 1: Delete control pod and wait for restart")
         resp = self.csm_obj.restart_control_pod(self.master)
-        assert_utils.assert_true(resp[0], resp[1])
+        assert resp[0], resp[1]
         # To get all the services up and running
```

Comment:
wherever u r sleeping make sure to add a logger that waiting for so and so  for  <time> so that test doesn't feel like stuck

Reply:
I am not even sure if we need this 40 seconds. This is old test. 
All i wanted to do is replace assert_utils where i can as traceback is wrong for them.